src/vision/status_quo/barbara_wishes_for_easy_runtime_switch.md

```diff
@@ -0,0 +1,59 @@
+# 😱 Status quo stories: Barbara wishes for an easy runtime switch
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from
+real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async
+Rust programmers face today.
+
+## The story
+
+[Barbara] has been working on an async codebase for the past 5 years. It is extremely mature and
+quite large (in the millions of lines of code). The product has traditionally been run on the server
+only, but now they want to move large parts to the edge. After some investigation, they realize that
+the open-source async runtime they have been using (runtime X), while great for server workloads, is
```

Comment:
This says "runtime X", but I think it's really about `tokio`...? We've generally tried to be real and concrete, rather than talking in hypotheticals. I think we should stick to that.

Reply:
This was my suggestion since I didn't want to make it seem like this is an issue with how tokio is designed, but as long as we make a mention in the FAQ that we see the same thing happening (to different degrees) with all runtimes, I'm happy with the idea of switching to use tokio in this example.

Reply:
I would definitely include a comment in the FAQ

Reply:
To me, referring "obliquely" to a runtime makes it read sinister or something.

Reply:
OK, so I'll change this to `tokio`.